YurisCodingClub/zero-minds
# [FEATURE] Create account view

## Description

This PR creates a user account component containing: 
- User's profile picture
- User's personal details
- User's skills
## Related Issue

closes #25 

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|     | :sparkles: New feature     |

## Updates

### Before

### After

![image](https://github.com/YurisCodingClub/zero-minds/assets/72811283/1a15e75e-440f-42a1-83c3-7a8789ca0e88)

## Testing Steps / QA Criteria

### To Test
Import the `UserAccount` component into any component and display it just before the last closing tag of the host component

For example, if you're using the gallery component as the host

Navigate to the `page.tsx` file under the gallery folder and in the file 

Add the following code at the top level
`import UserAccount from "@/components/UserAccount";`

Then above the `</main>` tag add `<UserAccount />`

Now type `npm run dev` in your terminal and click enter, the website will be rendered on your browser after a few seconds and the component should show up under the Gallery tab
